packages/ocal/ocal.0.2.2/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+maintainer: "Richard Mortier <mort@cantab.net>"
+authors: [ "Richard Mortier" ]
+license: "ISC"
+
+homepage: "https://github.com/mor1/ocal"
+dev-repo: "git+https://github.com/mor1/ocal.git"
+bug-reports: "https://github.com/mor1/ocal/issues"
+doc: "https://mor1.github.io/ocal/"
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+run-test: [
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```

Comment:
This doesn't work. It should be either tagged with `{with-test}` inside `build` or without inside `run-test`.

Reply:
Ah ok -- I think this was me misinterpreting the error from `dune-release lint` indicating what to do to remove the `build-test` field. Will fix!